README.md

```diff
@@ -2,38 +2,267 @@
 
 [![codecov](https://codecov.io/gh/Azure/notation-azure-kv/branch/main/graph/badge.svg)](https://codecov.io/gh/Azure/notation-azure-kv)
 
-Azure Provider for the Notary v2 [Notation CLI](https://github.com/notaryproject/notation)
+Azure Provider for the [Notation CLI](https://github.com/notaryproject/notation)
 
-## Getting Started:
-The following summarizes the steps to configure the Azure Key Vault notation plugin, configure gatekeeper, sign and verify a container image to Azure Kubernetes Service
+The notation-azure-kv plugin provides the capability to sign the Notation generated payload by using Azure Key Vault (AKV). The user's certificate and private key should be stored in AKV and the plugin will request signing and getting the leaf certificate from AKV. 
 
-```bash
-# Sign in with Azure CLI.
-# Other authorization methods are also available.
-# See https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authorizatio
-az login
+The plugin supports Azure CLI identity and Managed Identity for accessing AKV.
 
-# Add signing and verification keys to the notation configuration policy
-notation key add --name $KEY_NAME --plugin azure-kv --id $KEY_ID
-notation cert add --name $KEY_NAME $CERT_PATH
+## Installation
+Before installing notation azure key vault plugin, please make sure Notation CLI has been installed. If you didn't install it, please follow the [Notation installation guide](https://notaryproject.dev/docs/installation/cli/).
 
-# Install ratify, with the verification key
-helm install ratify ratify/charts/ratify \
-        --set registryCredsSecret=regcred \
-        --set ratifyTestCert=$PUBLIC_KEY
-kubectl apply -f ./ratify/charts/ratify-gatekeeper/templates/constraint.yaml
+1. Navigate to the [Releases](https://github.com/Azure/notation-azure-kv/releases) page and select the latest release of notation-azure-kv. The notation-azure-kv binaries for each platform are available under the Assets section. Please download it based on your own platform. 
+2. Validate the checksum which should be the same as the one in `checksums.txt` and then install the plugin.
+   
+   For Linux Bash:
+   ```sh
+   # validate checksum
+   sha256sum notation-azure-kv_<version>_linux_amd64.tar.gz
```

Comment:
It is suggested to do
```suggestion
   version=v0.6.0

   # validate checksum
   cat checksums.txt | grep notation-azure-kv_${version}_linux_amd64.tar.gz | sha256sum -c
```

Reply:
Again, use variables instead of placeholders.

Reply:
Updated.